test/e2e/e2e.bats

```diff
@@ -8,19 +8,17 @@ source ./test/helper/helper.sh
 @test "[e2e] maven task" {
     [ -n "${E2E_MAVEN_PARAMS_URL}" ]
     [ -n "${E2E_MAVEN_PARAMS_REVISION}" ]
+    [ -n "${E2E_MAVEN_PARAMS_SERVER_SECRET}" ]
+    
+    kubectl delete secret secret-maven || true
 
-    kubectl delete secret regcred || true
-    run kubectl create secret generic regcred \
-        --from-file=.dockerconfigjson=$HOME/.docker/config.json \
-        --type=kubernetes.io/dockerconfigjson
-    assert_success
-    run kubectl patch serviceaccount default -p '{"imagePullSecrets": [{"name": "regcred"}]}'
-    assert_success
```

Comment:
This part is/was actually required so that the image can be pulled from the github workflows. It is going to change with https://github.com/openshift-pipelines/task-maven/pull/10 though.